Remove the Cluster level keys info (#2119)
corda/corda-docs-portal: content/en/platform/corda/5.2/deploying-operating/keys/_index.md

   {{% /tab %}}
   {{< /tabs >}}

>    The system will take some time to propagate newly added master key across Corda. Corda will wrap any new managed wrapping keys with this new master wrapping key.

Reviewer: ```suggestion
   Corda will take some time to propagate the newly added master key across the cluster. Corda will wrap any new managed wrapping keys with this new master wrapping key.
```

### Rotating Managed Wrapping Keys

> Virtual node wrapping keys are managed by Corda and can be rotated using the POST method of the [/api/v5_2/wrappingkey/rotation/{tenantid} endpoint](../reference/rest-api/openapi.html#tag/Key-Rotation-API/operation/post_wrappingkey_rotation__tenantid_). Specify the following as the path parameter:

Reviewer: ```suggestion
Virtual node wrapping keys are managed by Corda and can be rotated using the POST method of the [/api/v5_2/wrappingkey/rotation/{tenantid} endpoint](../reference/rest-api/openapi.html#tag/Key-Rotation-API/operation/post_wrappingkey_rotation__tenantid_). Specify the short hash holding ID of the virtual node as the path parameter. For example:
```

> * The short hash holding ID of the virtual node to rotate a virtual node wrapping key.

Reviewer: ```suggestion
```

> For example, to rotate the Virtual Node wrapping key:

Reviewer: ```suggestion
```
